src/libawkward/array/ListArray.cpp

```diff
@@ -251,7 +251,7 @@ namespace awkward {
 
     int64_t carrylen = offsets.getitem_at_nowrap(offsets.length() - 1);
     Index64 nextcarry(carrylen);
-    struct Error err = util::awkward_listarray_broadcast_tooffsets64<T>(
+    struct Error err = kernel::listarray_broadcast_tooffsets64<T>(
```

Comment:
Same with this one: `..._broadcast_tooffsets_64` for both the kernel dispatch and the underlying kernel.